src/game.cpp

```diff
@@ -9087,7 +9099,7 @@ void game::reset_item_list_state(WINDOW *window, int height)
     tokens.push_back(_("<+/->Priority"));
 
     int gaps = tokens.size() + 1;
-    int letters = 0;
+    letters = 0;
```

Comment:
Hmm.  There's generally a big push in DDA code for signing one's ints.  What's the reasoning here?